operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -206,7 +207,22 @@ tekton_results_manifest(){
     --from-literal=S3_SECRET_ACCESS_KEY="$TEKTON_RESULTS_S3_PASSWORD" \
     -n tekton-results --dry-run=client -o yaml > "$results_s3_secret"
 
-    yq e -n '.resources += ["namespace.yaml", "tekton-results-secret.yaml", "tekton_results_s3.yaml"]' > "$results_kustomize"
+    cat <<EOF | kubectl apply -f - --dry-run=client -o yaml > "$results_minio_config"
```

Comment:
It would be better to have a template that is then modified by `yq` instead of dumping the file content in the script.

We can beautify this later.